src/Framework/Framework/Resources/Scripts/dotvvm-root.ts

```diff
@@ -132,11 +132,15 @@ if (compileConstants.debug) {
     (dotvvmExports as any).debug = true
 }
 
+
+
 declare global {
-    const dotvvm: typeof dotvvmExports & {debug?: true, isSpaReady?: typeof isSpaReady, handleSpaNavigation?: typeof handleSpaNavigation};
+    interface DotvvmGlobalExtensions {}
+
+    const dotvvm: DotvvmGlobalExtensions & typeof dotvvmExports & {debug?: true, isSpaReady?: typeof isSpaReady, handleSpaNavigation?: typeof handleSpaNavigation};
 
     interface Window {
-        dotvvm: typeof dotvvmExports
+        dotvvm: DotvvmGlobalExtensions & typeof dotvvmExports
```

Comment:
Not your mistake, but I'd guess we didn't want `const dotvvm` and `window.dotvvm` to have different types, would you mind to unify them (using `dotvvm: typeof dotvvm` could work)